samples/tfm_integration/psa_level_1/CMakeLists.txt

```diff
@@ -7,12 +7,19 @@ set(QEMU_KERNEL_OPTION "-device;loader,file=${CMAKE_BINARY_DIR}/tfm_qemu.hex")
 
 find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
 
+set(BL2 $ENV{BL2})
+if(NOT DEFINED BL2)
+    #by default build with MCUboot
+    set(BL2 True)
+endif()
```

Comment:
> I'm OK with this, but it will require updating the `mps2_an521_nonsecure` and `nucleo_l552ze_q` to set the BL2 status to true:
> 
> * https://github.com/zephyrproject-rtos/zephyr/blob/master/samples/tfm_integration/tfm_ipc/CMakeLists.txt#L11
> * https://github.com/zephyrproject-rtos/zephyr/blob/master/samples/tfm_integration/psa_level_1/CMakeLists.txt#L10

@microbuilder @tejlmand i've first set BL2 default True here and the other TFM sample.

Reply:
This can be removed, in TFM the BL2 is True when using `CFGFILE "ConfigRegressionIPC"` as is done in this file:
https://github.com/zephyrproject-rtos/zephyr/blob/7e70b77a78d3c565596b0bef11878748d7cfdff9/samples/tfm_integration/psa_level_1/CMakeLists.txt#L20

https://github.com/zephyrproject-rtos/trusted-firmware-m/blob/master/trusted-firmware-m/configs/ConfigRegressionIPC.cmake#L59-L61

Note: depends on this PR comment: https://github.com/zephyrproject-rtos/trusted-firmware-m/pull/12#discussion_r481984582

Reply:
@tejlmand thank you, i've removed this change, tested with zephyrproject-rtos/trusted-firmware-m#12 (comment)